scripts/settings.js

```diff
@@ -205,7 +205,7 @@ export default class Settings {
       Constants.MODULE_ID,
       Settings.FAVORITE_EFFECT_NAMES,
       {
-        name: 'Favorite Effect Names',
+        name: game.i18n.localize("Settings.FavoriteEffectNames"),

```

Comment:
In any setting where `config` is set to false, it doesn't need to be localized. They are values that only the module uses for data management.

Reply:
Noted